RedHat-UX/red-hat-design-system
# fix(tag): updating outline css to fix background

## What I did

1.  Updated the `rh-tag` stylesheet to fix the issues related to the `outline` variant styles not being applied for the background color.  


## Testing Instructions

1. Go to https://deploy-preview-1134--red-hat-design-system.netlify.app/elements/tag/demo/variants/
2. The tags should now all have the correct background color, text color, and stylings for the outline and filled variants.
